css/styles.css

```diff
@@ -0,0 +1,162 @@
+body {
+  background: rgb(208, 231, 239);
+  font-family: 'Times New Roman', Times, serif;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.container {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.row2 {
+  width: 100%;
+  text-align: center;
+}
+
+button {
+  color: gray;
+  cursor: pointer;
+  background-color: white;
+  width: 55px;
+  height: 55px;
+  border-radius: 50%;
+  border-style: none;
+  margin: 10px;
+}
+
+button:hover {
+  background: white;
+}
+
+#back-button {
+  text-decoration: none;
+  color: gray;
+  cursor: pointer;
+  background-color: white;
+  border-radius: 50%;
+  width: 45px;
+  height: 45px;
+  margin-right: 10px;
+  padding-top: 10px;
+  padding-left: 6px;
+}
+
+input {
+  border-style: none;
+  padding: 16px;
+  border-radius: 10px;
+}
+
+#favorites-button,
+#favorites-button2 {
+  color: rgb(177, 27, 27);
+  font-size: 25px;
+}
+
+#blue,
+#blue2 {
+  color: rgb(74, 144, 226);
+  font-size: 45px;
+  margin-left: 10px;
+  margin-bottom: 30px;
+}
+
+/* input {
+  padding: 10px;
+  font-size: 16px;
+  border: 2px solid #ccc;
+  border-radius: 5px;
+  margin-right: 10px;
+} */
```

Comment:
Delete commented out/unused code before you make pull requests. You want to make sure that the code going into your main branch is clean.